sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/Custom/BackupProtectedItemCollection.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.ResourceManager.RecoveryServicesBackup
+{
+    /// <summary> The BackupProtectedItemCollection. </summary>
```

Comment:
Let us remove the comment here - the compiler will just take the doc on the generated partial class.

Reply:
Fixed in latest commit.